proto/queue/queue.proto

```diff
@@ -23,6 +23,30 @@ message ControlEvent {
     ExecuteResponse execute_response = 3;
     DeliverablePrefixes deliverable_prefixes = 4;
     UndeliverablePrefixes undeliverable_prefixes = 5;
+    CloseJob close_job = 6;
+    Close close = 7;
```

Comment:
I assume we can change the close fieldnumber given that this isn't being used yet. Just double-checking

Reply:
Correct- these protos are only used in memory, single process, and not stored anywhere. It is the equivalent of changing a struct definition for an internal type.

We are using protos for these intnernal API types because proto is good.